templates/html/html.html.twig

```diff
@@ -14,7 +22,7 @@
     <css-placeholder token="{{ placeholder_token }}">
     <js-placeholder token="{{ placeholder_token }}">
   </head>
-  <body{{ attributes.addClass('language-' ~ html_attributes.lang).addClass('ecl-typography') }}>
+  <body{{ attributes.addClass('language-' ~ html_attributes.lang).addClass('ecl-typography').addClass(body_classes) }}>
```

Comment:
We could merge these two classes added here to the newly created `body_classes` array.